program/src/processor.rs

```diff
@@ -4796,6 +5071,140 @@ impl Processor {
         Ok(())
     }
 
+    /// *** Create a DustAccount PDA and initialize it
+    #[inline(never)]
+    fn create_dust_account(program_id: &Pubkey, accounts: &[AccountInfo]) -> MangoResult {
+        const NUM_FIXED: usize = 4;
+        let accounts = array_ref![accounts, 0, NUM_FIXED];
+        let [
+            mango_group_ai,         // read
+            mango_account_ai,       // write
+            payer_ai,               // write & signer
+            system_prog_ai,         // read
+        ] = accounts;
+        check!(
+            system_prog_ai.key == &solana_program::system_program::id(),
+            MangoErrorCode::InvalidProgramId
+        )?;
+        check!(payer_ai.is_signer, MangoErrorCode::SignerNecessary)?;
+
+        let _mango_group = MangoGroup::load_checked(mango_group_ai, program_id)?;
+        let rent = Rent::get()?;
+
+        let mango_account_seeds: &[&[u8]] = &[&mango_group_ai.key.as_ref(), b"DustAccount"];
+        // TODO - test passing in MangoAccount that already has some data in it
+        seed_and_create_pda(
+            program_id,
+            payer_ai,
+            &rent,
+            size_of::<MangoAccount>(),
+            program_id,
+            system_prog_ai,
+            mango_account_ai,
+            mango_account_seeds,
+            &[],
+        )?;
+        let mut mango_account: RefMut<MangoAccount> = MangoAccount::load_mut(mango_account_ai)?;
+
+        mango_account.mango_group = *mango_group_ai.key;
+        mango_account.owner = *mango_group_ai.key;
```

Comment:
I think the owner should be admin. If we ever needed to do anything with this account, it'd probably be via the DAO